index.html

```diff
@@ -53,46 +79,35 @@ <h2>I’am a Software Developer</h2>
                     </li>
                 </ul>
             </div>
+        </div>
+        <div class="img-cont"></div>
     </section>
     <section class="recent-works">
-        <h2 class="title">My Recent Works</h2>
-        <img class="vector" src="./images/Vector.png" alt="Vector">
-        <article>
-        <img class="themes" src="./images/Img Placeholder.png" alt="Placeholder">
-        <h3>Multi-Post Stories</h3>
-        <p>A daily selection of privately personalized<br> reads; no accounts or sign-ups required.<br> has been the industry's standard
-        dummy<br> text ever since the 1500s, when an<br> unknown printer took a standard dummy<br> text.</p>
-        <ul>
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-        </ul>
-        <button class="btn-multi" type="button">See Project</button>
+        <div class="recent">
+            <h2 class="title">My Recent Works</h2>
+            <img class="vector" src="./images/Vector.png" alt="Vector">
+        </div>
+        <article class="multi">
+            <div>
+                <img class="themes" src="./images/Img Placeholder.png" alt="Placeholder">
+            </div>
+            <div class="multi-post">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized<br> reads; no accounts or sign-ups required.<br> has been the industry's
+                    standard
+                    dummy<br> text ever since the 1500s, when an<br> unknown printer took a standard dummy<br> text.</p>
```

Comment:
- [OPTIONAL] :recycle:  Kindely make sure you use the same font('Inter') as per the the figma design as recommended by the project requirements

- Your design:
![YourText](https://user-images.githubusercontent.com/97834160/206588358-cf42f47f-69b8-4fa9-b130-57c217169c9b.png)

- Figma:
![FigText](https://user-images.githubusercontent.com/97834160/206588398-c77be1e6-e646-43da-a9d0-fa7f4cbc4da6.png)